app/client/templates/views/account.html

```diff
@@ -161,11 +161,24 @@ <h2 class="copyable-address">{{walletIcon}}
                     {{> elements_transactions_table ids=transactions account=_id}}
                 {{/if}}
 
-                {{#if contractEvents}}
+                {{#with customContract}}
                     <h2>{{i18n 'wallet.events.latest'}}</h2>
-
+                    <br>
```

Comment:
please remove the code from your other PR. Or we can wait until that one merges.